sphinx/tutorial/rose/suites.rst

```diff
@@ -508,217 +508,3 @@ Rose Applications In Rose Suite Configurations
 
    Or alternatively using :envvar:`ROSE_TASK_APP`.
 
-
```

Comment:
For now could we put this tutorial back but replace "Rose Bush" with "Cylc Review", I guess we'll move it into the Cylc tutorial at some point.

Reply:
I feel that would be off topic in a rose tutorial.

Reply:
Its a Rose tutorial showing users how Rose works with Cylc suites...